flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/MultipleInputSelectionHandler.java

```diff
@@ -112,7 +112,7 @@ int selectNextInputIndex(int lastReadInputIndex) {
 	}
 
 	boolean shouldSetAvailableForAnotherInput() {
```

Comment:
meta note: it's really confusing to have inputSelection, InputSelectable, and inputSelector. Could you try to consolidate that?

Reply:
Do you have some proposal? `InputSelectable` is an interface for users to implement, to provide the chosen `InputSelection`. That gives as `inputSelector` from our perspective. Now we are wrapping some input selection handling logic into classes like `MultipleInputSelectionHandler`. How would you suggest to change it?

Reply:
Can't we keep the name of `InputSelectable` `inputSelectable`? It's especially tricky to distinguish `inputSelector` and `inputSelection` (edit distance 2).

Reply:
`inputSelectable ` is consistent but It's a bit worse description what it does. But ok, I can change it if you prefer so. (will be a commit on top)

Reply:
then please have the extra commit and I will check if readability improves for me. I also got your point, but I was really confused initially.

Reply:
Ops sorry, I deleted my comment and replaced it with just a thumb up, as I changed my mind. I like the `inputSelectable` name for this field better :)